src/OrbitBase/CMakeLists.txt

```diff
@@ -71,30 +72,29 @@ target_sources(OrbitBase PRIVATE
         WhenAll.cpp
         WriteStringToFile.cpp)
 
-if (WIN32)
-target_sources(OrbitBase PRIVATE
-        include/OrbitBase/GetLastError.h
-        include/OrbitBase/GetProcAddress.h)
+if(WIN32)
+        target_sources(OrbitBase PRIVATE
+                include/OrbitBase/GetLastError.h
+                include/OrbitBase/GetProcAddress.h)
 
-target_sources(OrbitBase PRIVATE
-        ExecutablePathWindows.cpp
-        GetLastErrorWindows.cpp
-        GetProcAddressWindows.cpp
-        ThreadUtilsWindows.cpp)
+        target_sources(OrbitBase PRIVATE
+                ExecutablePathWindows.cpp
+                GetLastErrorWindows.cpp
+                GetProcAddressWindows.cpp
+                ThreadUtilsWindows.cpp)
 else()
-target_sources(OrbitBase PRIVATE
-        ExecutablePathLinux.cpp
-        ExecuteCommandLinux.cpp
-        GetProcessIdsLinux.cpp
-        ThreadUtilsLinux.cpp)
+        target_sources(OrbitBase PRIVATE
+                ExecutablePathLinux.cpp
+                ExecuteCommandLinux.cpp
+                GetProcessIdsLinux.cpp
+                ThreadUtilsLinux.cpp)
 endif()
 
 target_link_libraries(OrbitBase PUBLIC
         CONAN_PKG::abseil
         CONAN_PKG::outcome
         std::filesystem)
 
-
```

Comment:
These are unrelated changes and shouldn't be part of the PR

Reply:
Argh. Sorry. Autoformat.